### ISSUE TITLE
Quail does not respect scope without guideline

### ISSUE THREAD

**Issue by mlewand** [quailjs/quail#286]
# Overview

It seems that Quail won't respect the scope given by jQuery selector if we won't provide `guideline` property in config.

We can see it with following [sample](https://gist.github.com/mlewand/ffad3a5ddc6bb5cc7751). Just download linked file to your `quail` directory.
## What it does?

It runs the Quail with a following call: `$( '#sampleContent' ).quail( { ... } )`, so that means that Quail should look **only** in #sampleContent element.

In given sample we're looking only for `imgHasAlt` test.

If no `guideline` property is given in Quail config, all tests are ran. But in this particular case it will run against wole document.

Just to compare other situation, uncomment the `guideline` property, and it works as expected.

**Comment by jessebeach** [comment]
Good catch. I'll see if this can't be addressed in the #289 branch I'm working on.

**Comment by theodoreb** [comment]
Related to #255 ?
